src/features/game/events/landExpansion/feedBumpkin.ts

```diff
@@ -33,12 +33,18 @@ export function feedBumpkin({
   if (quantity.lte(0)) {
     throw new Error("You have none of this food type");
   }
-
+  const { skills, equipped } = bumpkin;
+  const { tool } = equipped;
   inventory[action.food] = quantity.sub(1);
 
   let foodExperience = CONSUMABLES[action.food].experience;
 
-  if (bumpkin.skills["Kitchen Hand"]) {
+  if (skills["Kitchen Hand"]) {
```

Comment:
This will probably conflict with https://github.com/sunflower-land/sunflower-land/pull/1693 

So i will wait until that PR gets merged to fix it.

Reply:
Conflicts solved!